scripts/install/install-windows.bat

```diff
@@ -1,2 +1,2 @@
 @echo off
-PowerShell.exe -ExecutionPolicy Bypass -Command "%~dp0\install-windows.ps1 %*"
+PowerShell.exe -ExecutionPolicy Bypass -Command "& '%~dp0/install-windows.ps1' %*"
```

Comment:
Won't the & here make the script non-blocking? (Excuse my lack of Powershell expertise)

Reply:
Thats what you would think, but its powershell. Basically just means run the command in powershell, equivalent roughly to  Invoke-Expression